typescript-conditionals/main.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+function isOldEnoughToDrink(person) {
+  if (person.age < 21) {
+    return false;
+  } else {
+    return true;
+  }
+}
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age && person.name) {
```

Comment:
They just want you to combine the 2 conditions (person.age >= 21 and person.age >= 16) using and AND operator: &&

Reply:
Ah, I assumed it should've been set to false no matter what so I put it for if the property has any value in place.